components/Text/Text.tsx

```diff
@@ -76,8 +76,15 @@ export const Text = styled('span', {
         textTransform: 'uppercase',
       },
       capitalize: {
-        textTransform: 'capitalize',
+        // WARNING: this will only work with block elements (display block/inline-block)
+        // @see https://developer.mozilla.org/en-US/docs/Web/CSS/::first-letter
```

Comment:
:question: should I enforce a `display` value for this variant to secure the behaviour ?